backend/composer/api/urls.py

```diff
@@ -42,4 +43,5 @@
 urlpatterns = [
     path("", include(router.urls)),
     path("jsonschemas/", jsonschemas, name="jsonschemas"),
+	path("knowledge-statement/", KnowledgeStatementViewSet.as_view(), name="knowledge-statement"),
```

Comment:
question: @D-GopalKrishna is there a special reason why you didn't add it to the DefaultRouter (created on line 20)?